modules/integration_aws-elasticsearch/detectors-elasticsearch.tf

```diff
@@ -121,7 +123,10 @@ resource "signalfx_detector" "cpu_90_15min" {
   tags                    = compact(concat(local.common_tags, local.tags, var.extra_tags))
 
   program_text = <<-EOF
-    signal = data('CPUUtilization', filter=filter('namespace', 'AWS/ES') and filter('stat', 'upper') and filter('NodeId', '*') and ${module.filtering.signalflow})${var.cpu_90_15min_aggregation_function}${var.cpu_90_15min_transformation_function}.publish('signal')
+    data_node_cpu   = data('CPUUtilization',       filter=filter('namespace', 'AWS/ES') and filter('stat', 'upper') and filter('NodeId', '*') and ${module.filtering.signalflow})${var.cpu_90_15min_aggregation_function}${var.cpu_90_15min_transformation_function}
+    warm_node_cpu   = data('WarmCPUUtilization',   filter=filter('namespace', 'AWS/ES') and filter('stat', 'upper') and filter('NodeId', '*') and ${module.filtering.signalflow})${var.cpu_90_15min_aggregation_function}${var.cpu_90_15min_transformation_function}
+    master_node_cpu = data('MasterCPUUtilization', filter=filter('namespace', 'AWS/ES') and filter('stat', 'upper') and filter('NodeId', '*') and ${module.filtering.signalflow})${var.cpu_90_15min_aggregation_function}${var.cpu_90_15min_transformation_function}
```

Comment:
AWS documentation recommends different alert rules for master nodes than classic/UltraWarm nodes regarding CPU utilization:

> Alarm: CPUUtilization or WarmCPUUtilization maximum is >= 80% for 15 minutes, 3 consecutive times
> Issue: 100% CPU utilization isn't uncommon, but sustained high usage is problematic. Consider using larger instance types or adding instances.

> Alarm: MasterCPUUtilization maximum is >= 50% for 15 minutes, 3 consecutive times
> Issue: Consider using larger instance types for your dedicated master nodes. Because of their role in cluster stability and blue/green deployments, dedicated master nodes should have lower CPU usage than data nodes.

I believe we should create a separate detector for UltraWarm nodes.

Reply:
done

Reply:
"15 minutes, 3 consecutive times" so I just set "20 minutes"